ext/native/thin3d/thin3d_vulkan.cpp

```diff
@@ -1277,18 +1350,124 @@ uint32_t VKContext::GetDataFormatSupport(DataFormat fmt) const {
 	}
 }
 
+// Simple independent framebuffer image. Gets its own allocation, we don't have that many framebuffers so it's fine
+// to let them have individual non-pooled allocations.
```

Comment:
I wonder if it's worth logging it.  I could imagine some crazy game using a bunch of tiny FBOs for shadows or something.  Though probably still not enough that it matters...

-[Unknown]